tests/testthat/test-calculate_stats.R

```diff
@@ -0,0 +1,32 @@
+test_that("calculate_stats works", {
+  skip_on_cran()
+  skip_if_offline("github.com")
+
+  s1 <- calculate_stats(seasons = 2023, summary_level = "season", stat_type = "player")
+  s2 <- calculate_stats(seasons = 2023, summary_level = "week",   stat_type = "player")
+  s3 <- calculate_stats(seasons = 2023, summary_level = "season", stat_type = "team")
+  s4 <- calculate_stats(seasons = 2023, summary_level = "week",   stat_type = "team")
+  s5 <- calculate_stats(seasons = 2023, summary_level = "week",   stat_type = "player", season_type = "POST")
+
+  n1 <- names(s1)
+  n2 <- names(s2)
+  n3 <- names(s3)
+  n4 <- names(s4)
+  n5 <- names(s5)
+
+  var_names <- nflfastR::nfl_stats_variables$variable
+
+  # Make sure variable names are listed in nflfastR::nfl_stats_variables$variable
+  expect_in(n1, var_names)
+  expect_in(n2, var_names)
+  expect_in(n3, var_names)
+  expect_in(n4, var_names)
+  expect_in(n5, var_names)
+
```

Comment:
we could snapshot the schema and type as well e.g. make sure skimr::skim() is consistent for field names and values? 

skim <- skimr::skim(new_stats) %>% 
    as.data.frame() %>% 
    dplyr::transmute(variable = skim_variable, type = skim_type, n_missing = round(n_missing, -1))

expect_snapshot_value(skim)

Reply:
done in e1f357fa0d55c96bb7944e5f1ed26edd0bee85d2